docs/source/jupyter/ontology-interface.ipynb

```diff
@@ -107,101 +162,415 @@
    "metadata": {
     "tags": []
    },
+   "outputs": [],
+   "source": [
+    "from osp.core.namespaces import city\n",
+    "from osp.core.namespaces import math"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are several ways to access an ontology entity:\n",
+    "\n",
+    "- By **suffix**. For example, for the namespace `city`, whose [IRI](https://fusion.cs.uni-jena.de/fusion/blog/2016/11/18/iri-uri-url-urn-and-their-differences/) is `http://www.osp-core.com/city#`, fetching by the suffix `Citizen` would return the ontology entity with IRI `http://www.osp-core.com/city#Citizen`.\n",
+    "\n",
+    "- By **label**. Fetchs the entity by the label that has been assigned to it using either the `rdfs:label` or `skos:prefLabel` predicates.\n",
+    "\n",
+    "- By **IRI**. The full [IRI](https://fusion.cs.uni-jena.de/fusion/blog/2016/11/18/iri-uri-url-urn-and-their-differences/) of an ontology entity is provided in order to fetch it.\n",
+    "\n",
+    "- By **string**. Using a string, for example `\"city.LivingBeing\"`. This is only useful in some special cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The **most convenient way** to access an ontology entity is using the **dot notation** in python, i.e. `city.Citizen`. This method is a shorthand for fetching by suffix or label:  \n",
+    "\n",
+    "- When the keyword `reference_by_label` is set to `True` (enabled) in the [ontology YAML installation file](../yaml.md), the dot notation is a shorthand for fetching by label. This keyword is **enabled** in the `math` namespace.\n",
```

Comment:
it seems the wrong link is given: `../owl.html` is what is meant perhaps (rather then `../yaml.md`)

Reply:
I meant `../yaml.md` but I realized that the keyword is not even documented there, plus the example is about the EMMO which is an OWL ontology. So yes, we might be better off using `../owl.html` that at least contains the keyword and matches the example.